Game/MyProject/IvGraphics/IvVertexFormats.h

```diff
@@ -36,10 +36,8 @@ struct VertexAttribute
 
 class VertexDescription
 {
-    std::vector<VertexAttribute> attributes;
-    uint32_t vertexSize = 0;
 public:
-    VertexDescription() = default;
+    VertexDescription() {};
```

Comment:
VertexDescription() = default; and VertexDescription() {}; are basically the same thing.